.github/workflows/ci.yml

```diff
@@ -10,19 +10,8 @@ env:
   YARN_ENABLE_GLOBAL_CACHE: false
 
 jobs:
-  commit-lint:
```

Comment:
Why remove commit lint, you need to rebase

Reply:
I forgot to update the main branch, so sorry, this linti crushed it 😢

Reply:
ı solved